module.flow.js

```diff
@@ -76,6 +76,11 @@ declare module '@solana/web3.js' {
       callback: AccountChangeCallback,
     ): number;
     removeAccountChangeListener(id: number): Promise<void>;
+    onProgramAccountChange(
+      programIdL PublicKey,
```

Comment:
looks like a typeo here, unfortunately we don't currently test module.flow.js in CI and rely on downstream packages to notify of bustage 🙃

Reply:
😝